docs/migration-guide/to-15.md

```diff
@@ -36,30 +36,60 @@ Additionally, you may no longer need to extend [Prettier's Stylelint config](htt
+
+There are lots of other rules we don't turn on in the [standard config](https://www.npmjs.com/package/stylelint-config-standard) and you can learn more about using them to customize Stylelint to your exact needs in our [new guide](../user-guide/customize.md).
+
+If you want to continue using Stylelint to enforce stylistic consistency, you can [migrate the deprecated rules you need to a plugin](../developer-guide/plugins.md).
 
 ## Breaking changes
 
 Three breaking changes may also affect you:
 
+- removed `processors` configuration property
 - removed support for Node.js 12
-- removed support for processors
 - changed `overrides.extends` behavior
 
+### Removed `processors` configuration property
+
+Processors were our first attempt to support containers of CSS, e.g. Markdown, HTML and CSS-in-JS. We later introduced [custom syntaxes](../developer-guide/syntaxes.md) to fix some of the shortcomings of processors, e.g. incompatibility with the [`--fix` option](../user-guide/options.md#fix).
+
+We've also deprecated the [`postcss-css-in-js`](https://www.npmjs.com/package/@stylelint/postcss-css-in-js) custom syntax. It was not possible to maintain a monolithic custom syntax that attempted to support every CSS-in-JS library, as there are so many of them and each with variations in syntax.
+
+You should remove the `processors` property from your configuration object and use a library-specific [custom syntax](../user-guide/options.md#customsyntax) instead.
+
+For example, if you use [styled-components](https://styled-components.com):
+
+```diff json
+{
+- "processors": ["stylelint-processor-styled-components"],
++ "customSyntax": "postcss-styled-components",
+  "rules" { .. }
+}
+```
+
+Other custom syntaxes include:
+
+- [postcss-html](https://www.npmjs.com/package/postcss-html)
+- [postcss-markdown](https://www.npmjs.com/package/postcss-markdown)
+- [postcss-lit](https://www.npmjs.com/package/postcss-lit)
+- [@linaria/postcss-linaria](https://www.npmjs.com/package/@linaria/postcss-linaria)
+
+You'll find a complete list of them in [Awesome Stylelint](https://github.com/stylelint/awesome-stylelint#readme)
+
+If you create a new custom syntax, please [open a pull request](https://github.com/stylelint/awesome-stylelint/compare) to update [Awesome Stylelint](https://github.com/stylelint/awesome-stylelint#readme) so that others can find it. For example, [Stitches](https://stitches.dev/) and [Vanilla Extract](https://vanilla-extract.style/) need syntaxes, which are object-based CSS-in-JS libraries.
+
 ## Removed support for Node.js 12
 
-You should use the following or higher versions of Node.js:
+Node.js 12 has reached end-of-life. We've removed support for it so that we could update some of our dependencies. You should use the following or higher versions of Node.js:
 
 - 14.13.1
 - 16.0.0
```

Comment:
IMHO it should be TLS only.
i.e.
```suggestion
- 16.13.0 
- 18.12.0
```

Reply:
This seems to follow `engines.node` in `package.json`:

https://github.com/stylelint/stylelint/blob/f129f3b992accbf37c7d532d0d020c62c4f3749a/package.json#L197-L199

IMHO, I don't think it is necessary to support versions strictly.

Reply:
It's probably off topic but it should probably be `^14.15.0 || >=16.13.0`.
i.e. Fermium or GTE Gallium

ref https://nodejs.org/en/download/releases/